src/replays/replayitem.py

```diff
@@ -105,175 +90,194 @@ def __init__(self, uid, parent, *args, **kwargs):
     
     def update(self, message, client):
-        '''
-        Updates this item from the message dictionary supplied
-        '''
-        
+        # Updates this item from the message dictionary supplied #
         
-        self.client  = client
+        self.client = client
         
-        self.name       = message["name"]
-        self.mapname    = message["map"]
-        self.duration   = time.strftime('%H:%M:%S', time.gmtime(message["duration"]))
-        self.startHour  = time.strftime("%H:%M", time.localtime(message['start']))
-        self.startDate  = time.strftime("%Y-%m-%d", time.localtime(message['start']))
-        self.mod        = message["mod"]
-         
+        self.name      = message["name"]
+        self.mapname   = message["map"]
+        self.duration  = time.strftime('%H:%M:%S', time.gmtime(message["duration"]))
+        self.startHour = time.strftime("%H:%M", time.localtime(message['start']))
+        self.startDate = time.strftime("%Y-%m-%d", time.localtime(message['start']))
+        self.mod       = message["mod"]
+
         # Map preview code
         self.mapdisplayname = maps.getDisplayName(self.mapname)
       
         self.icon = maps.preview(self.mapname)
         if not self.icon:
             self.client.downloader.downloadMap(self.mapname, self, True)
             self.icon = util.icon("games/unknown_map.png")        
-        #self.setIcon(0, self.icon)
-        
-        self.moddisplayname = self.mod
-        self.modoptions = []
 
-        if self.mod in mods :
+        if self.mod in mods:
             self.moddisplayname = mods[self.mod].name 
+        else:
+            self.moddisplayname = self.mod
 
 #        self.title      = message['title']
-#        self.host       = message['host']
 #        self.teams      = message['teams']
 #        self.access     = message.get('access', 'public')
 #        self.mod        = message['featured_mod']
+#        self.host       = message["host"]
 #        self.options    = message.get('options', [])
 #        self.numplayers = message.get('num_players', 0) 
 #        self.slots      = message.get('max_players',12)
 
-        self.viewtext = (self.FORMATTER_REPLAY.format(time=self.startHour, name=self.name, map = self.mapdisplayname, duration = self.duration, mod = self.moddisplayname))
+        self.viewtext = self.FORMATTER_REPLAY.format(time=self.startHour, name=self.name, map=self.mapdisplayname,
+                                                     duration=self.duration, mod=self.moddisplayname)
 
     def infoPlayers(self, players):
-        '''
-        processes information from the server about a replay into readable extra information for the user, also calls method to show the information
-        '''
+        # processes information from the server about a replay into readable extra information for the user, #
+        # also calls method to show the information #
+
         self.moreInfo = True
+        self.numberplayers = len(players)
```

Comment:
Certainly `len(players)` is more descriptive than an instancefield called `numberplayers`.